base_layer/wallet_ffi/wallet.h

```diff
@@ -272,7 +272,15 @@ void comms_config_destroy(struct TariCommsConfig *wc);
 /// -------------------------------- TariWallet ----------------------------------------------- //
 
 // Creates a TariWallet
-struct TariWallet *wallet_create(struct TariWalletConfig *config, char *log_path, int* error_out);
+struct TariWallet *wallet_create(struct TariWalletConfig *config,
+                                    char *log_path,
+                                    void (*call)(struct TariPendingInboundTransaction*),
+                                    void (*call)(struct TariCompletedTransaction*),
+                                    void (*call)(struct TariCompletedTransaction*),
+                                    void (*call)(struct TariCompletedTransaction*),
+                                    void (*call)(struct TariCompletedTransaction*),
+                                    void (*call)(ulonglong, bool),
```

Comment:
ulonglong should be unsigned long long, or unsigned long long int (they are equivalent).

Reply:
done